docker-compose.yml

```diff
@@ -15,14 +15,42 @@ services:
     restart: always
     build: .
     volumes:
-      - .:/app
-      - /app/node_modules
+      - ./server/:/app/server
     environment: 
       MONGO_URI_DEV: mongodb://mongo:27017/reviews
     ports:
       - "5002:5002"
     depends_on: 
       - mongo
 
+  pg:
+    image: postgres
+    container_name: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
```

Comment:
Also not sure- do you want to include the un/pw in what is pushed to github?

Reply:
Same reasons from my comment above